src/store/token.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+export async function getToken(token) {
+	const tokenValue = await ((typeof (token) === 'function')
+		? token()
+		: token);
+
+	return await new Token(tokenValue);
+}
+
+export function compareTokens(a, b) {
```

Comment:
Using the word "compare" makes me think of a comparison function that returns an integer (e.g. 0, < 0, > 0) like what would be used with the `sort()` method of an array. However, this function returns a boolean since it's just an equality check perhaps use the term "equals" instead of "compare". Perhaps model this after Java's `Object.equals()` method and make this a member method of the `Token` class.

Reply:
Change of name I agree. Compare doesn't make sense.

I thought object.equals wouldn't be value based but it is! so I can change it to that.